config/beanstalk.clean.yml

```diff
@@ -1,6 +1,6 @@
 beanstalkd:
-  host: ~
-  port: ~
+  addresses:
```

Comment:
non-blocking comment: at some point we should probably figure out how to encapsulate the issue of multiple beanstalk servers from the workers.  If this ever gets misconfigured (a worker is watching on one address but its counterpart is submitting on another) it's going to be tough to debug.

Reply:
Good point. In that process we should also look at alternative message passing frameworks like Kafka that have high availability built in.

Reply:
sgtm, I'd be curious to hear your thoughts when we cross that bridge but would defer to your expertise on the choice